src/Host.cpp

```diff
@@ -877,14 +878,8 @@ bool Host::installPackage(const QString& fileName, int module)
 
                 if (!fd.open(QIODevice::ReadWrite | QIODevice::Truncate)) {
                     //FIXME: report error to user
-                    qDebug() << "Host::installPackage("
-                             << fileName
-                             << ","
-                             << module
-                             << ")\n    ERROR opening:"
-                             << QStringLiteral( "%1%2" ).arg(_dest, entryInArchive)
-                             << "!\n    Reported error was:"
-                             << fd.errorString();
+                    qDebug() << "Host::installPackage(" << fileName << "," << module << ")\n    ERROR opening:" << QStringLiteral("%1%2").arg(_dest, entryInArchive)
```

Comment:
Yeah that's that I felt - it became hard to read when it was split over multiple lines.